### ISSUE TITLE
Itunes podcast subscription issue

### ISSUE THREAD

**Issue by gerroon** [fatg3erman/RompR#51]
Hi 

I was testing the Itunes search result fix. I am able to subscribe to some but not some others when I tried some searches. Here is one that does not subscribe with error.

It seems to me that the xml exists, so the subscription should happen but it throws and error. I am not sure if this is an issue with Rompr or something about the podcast , maybe I just hit an edge case when testing it.

```
Searching iTunes for podcasts 'acik%20gazete'
00:04:14 :     6744 : URL_DOWNLOADER      Downloading https://itunes.apple.com/search?term=acik%20gazete&entity=podcast
00:04:15 :     6744 : URL_DOWNLOADER        ..  Download Success
00:04:15 :     6744 : PODCASTS            Search found podcast : Açık Gazete
00:04:16 :     6744 : PODCASTS            Refreshing podcast 319
00:04:16 :     6744 : PODCASTS              Podcast title is Açık Gazete
00:04:16 :     6744 : URL_DOWNLOADER      Downloading http://acikradyo.com.tr/i/rss/Acik_Gazete.xml
00:04:17 :     6744 : URL_DOWNLOADER        ..  Download Success
00:04:17 :     6744 : PODCASTS            Image is http://acikradyo.com.tr/sites/default/files/acikradyo_600X600.jpeg
00:04:17 :     6744 : PODCASTS              Category is News & Politics
00:04:17 :     6744 : PODCASTS            Error 8 A non well formed numeric value encountered in /var/www/rompr/includes/podcastfunctions.php at line 175

```

**Comment by fatg3erman** [comment]
*Sighs and bangs head on table*

There's s standard for podcast rss feeds. It's not hard and it's not complicated. So why do so many people do it so wrong?

Every episode for a podcast should have a duration field which must be of the format 'HH:MM:SS' or 'MM:SS'. One (and only one) episode in the particular podcast has the duration field set to 'Açık Gazete: 25 Ocak 2018'. I mean how does that even happen?

I'll have to put in a check that the duration is actually a number.

**Comment by gerroon** [comment]
I am sorry man, I am just a messenger. I actually subscribed to that one on my phone phone using Antennapod, so I assumed the podcast was good already on that front. 

I think that people are sloppy and I see that all the time with podcasts, sometimes the mediaa links even do not play :( let alone get the timing correct.

**Comment by fatg3erman** [comment]
Yes the timing is very often inaccurate. There is also a field for the file size (in bytes) which is completely wrong in every podcast I've ever examined. And many other fields too aren't used correctly, parsing podcast feeds is a minefield!

Thanks for all your testing, it's good to find bugs, sometimes frustrating when it's down to sloppy syndication software though!

**Comment by gerroon** [comment]
This issue explains why I was not seeing any updates from that podcasts from the start right after I imported my opml, it probably never imported the podcast in the first place due to this error.

I am happy to test as much as I can. You did a splendid job with this software, I am not sure why more people are not using it. We might need to do some plug work and promo work.
